### ISSUE TITLE
Advanced Pressure Gauge : Greater / Lower Than sliders don't 'stick' to mouse properly

### ISSUE THREAD

**Issue by wrincewind** [TeamPneumatic/pnc-repressurized#183]
So this is ... sort of a feature request?
I noticed when using the advanced GUI to tweak redstone output from e.g. the pressure gauge (or any of the other tube components you can apply an advanced PCB to), the sliders have somewhat unexpected behaviour.

Basically, if you drag one of the two sliders up and down, it works as you'd expect... unless you mouse over the other slider. Then that other slider jumps to your current mouse position and the first slider stops moving, as though you'd let go of one slider and grabbed onto the other. Optimally, I'd expect it to act like sliders in most other programs, where only the slider you're currently dragging is effected by mouse position.

**Comment by desht** [comment]
Yeah, that would make more intuitive sense, I agree.  Consider it a quality-of-life improvement :)  Shouldn't be too hard to implement I think.
